server/src/controllers/UserController.js

```diff
@@ -279,6 +279,37 @@ const UserController = {
       next(err);
     }
   },
+
+  /**
+   * Checks to see if a user exists with that email
+   * @param {Object} req
+   * @param {Object} res
+   * @param {Function} next
+   * @return {Promise<void>}
+   */
+  async userExist(req, res, next) {
+    const email = req.body?.email;
+
+    try {
+      const existingUser = await UserServices.getUserByEmail(email);
+      if (existingUser) {
+        console.log('FOUND');
+        console.log(existingUser);
+        return res.status(200).send({
+          message: 'User exists',
+          code: 1,
+        });
+      } else {
+        return res.status(200).send({
+          message: 'User does not exist',
+          code: 0,
+        });
```

Comment:
Can you use the `res.status` to indicate whether or not the user was found? The standard http code for that is 404

Reply:
Addressed here: https://github.com/UofT-Frosh-Orientation/orientation-website/pull/573/commits/b7c05fdf2e97c28b83faef0d19907527139952e5#diff-adba94941d312d5cbd213b59db6bac408d3edf3dff2d7d17752274382516ce73R387